### ISSUE TITLE
[env] Can't add numeric env value

### ISSUE THREAD

**Issue by arranw** [architect/architect#1336]
**Describe the issue**
When adding an environment variable using `arc env`, numeric values throw an error.

**Steps to reproduce**
1. Use the command `arc env --add --env staging PROJECT_ID "10000"`
2. Error from `aws-sdk`: `env failed! Expected params.Value to be a string`

**Expected behavior**
The input value is interpreted as a string and the environment variable is set as normal

**Screenshots**
If applicable, add screenshots to help explain your problem
```
 arc env --add --env staging PROJECT_ID "10000"
         App ⌁ app-name
      Region ⌁ ca-central-1
     Profile ⌁ default
     Version ⌁ Architect 10.2.2
         cwd ⌁ /home/arranw/projects/app-name

env failed! Expected params.Value to be a string
InvalidParameterType: Expected params.Value to be a string
    at ParamValidator.fail (/home/arranw/.config/yarn/global/node_modules/aws-sdk/lib/param_validator.js:50:37)
    at ParamValidator.validateType (/home/arranw/.config/yarn/global/node_modules/aws-sdk/lib/param_validator.js:232:10)
    at ParamValidator.validateString (/home/arranw/.config/yarn/global/node_modules/aws-sdk/lib/param_validator.js:154:32)
    at ParamValidator.validateScalar (/home/arranw/.config/yarn/global/node_modules/aws-sdk/lib/param_validator.js:130:21)
    at ParamValidator.validateMember (/home/arranw/.config/yarn/global/node_modules/aws-sdk/lib/param_validator.js:94:21)
    at ParamValidator.validateStructure (/home/arranw/.config/yarn/global/node_modules/aws-sdk/lib/param_validator.js:75:14)
    at ParamValidator.validateMember (/home/arranw/.config/yarn/global/node_modules/aws-sdk/lib/param_validator.js:88:21)
    at ParamValidator.validate (/home/arranw/.config/yarn/global/node_modules/aws-sdk/lib/param_validator.js:34:10)
    at Request.VALIDATE_PARAMETERS (/home/arranw/.config/yarn/global/node_modules/aws-sdk/lib/event_listeners.js:132:42)
    at Request.callListeners (/home/arranw/.config/yarn/global/node_modules/aws-sdk/lib/sequential_executor.js:106:20)
```

**Desktop**
Please complete the following information (if appropriate):
 - OS: WSL 2 / Ubuntu 20.04
 - Version: Architect 10.2.2

**Additional context**

**Comment by ryanblock** [comment]
Gross, looks like Minimist is probably detecting that as a number and re-coercing it back, so we'll need to always string normalize those passed args. Thanks!

**Comment by ryanblock** [comment]
Oh sorry, this shipped in July in 10.3.4!
